docs/src/main/tut/json.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title: JSON handling
+---
+
+## Pick a library
+
+### Argonaut
+
+```scala
+libraryDependencies += Seq(
+  "org.http4s" %% "http4s-argonaut" % "0.15.0a-SNAPSHOT",
+  "com.github.alexarchambault" %% "argonaut-shapeless_6.1" % "1.1.1" // or 1.1.0 for non-a versions
+)
+```
+
+### Circe
+
+```scala
+libraryDependencies += "org.http4s" %% "http4s-circe" % "0.15.0a-SNAPSHOT"
+```
+
+### Json4s
+
+```scala
+libraryDependencies += "org.http4s" %% "http4s-json4s" % "0.15.0a-SNAPSHOT"
+```
+
+And one of
+
+```scala
+libraryDependencies += "org.http4s" %% "http4s-json4s-native" % "0.15.0a-SNAPSHOT"
+libraryDependencies += "org.http4s" %% "http4s-json4s-jackson" % "0.15.0a-SNAPSHOT"
+```
+
+## Import it
+
+The namespace is
+
+```scala
+import org.http4s.<libraryName>._
+```
+
+where `libraryName` can be one of the above, e.g. `org.http4s.argonaut._`.
+
+## Deriving codecs
+
+To use the functions provided by http4s, the corresponding implicit instances
+need to be in scope. For argonaut, there is [argonaut-shapeless], circe has the
+[circe-generic] module. With json4s, there's [jsonExtract], with the
+corresponding drawbacks.
```

Comment:
Oops.  That's what I get for reviewing the markdown instead of the previewed site.